snap/get_version

```diff
@@ -38,35 +38,32 @@ annotate_with_dirty() {
     echo "${version}"
 }
 
-# strip_branch_tag_prefix will remove the branch name prefix from the tag name.
-# $1: tag: the tag name to strip the prefix from.
-# $2: current_branch: the branch name to strip from the tag.
+# strip_branch_tag_prefix removes any non-numeric prefix ending with a
+# dash (e.g. "msentraid-") from the tag name. We use this to remove the
+# branch name prefix from the tag name, but we do not just strip the
+# current branch name because we also want to support branching of a new
+# branch and use the latest tag from that branch (for example when
+# branching of the msentraid branch to test a fix, then that branch
+# should still use a valid version). $1: tag: the tag name to strip the
```

Comment:
nitpick: I would align $1 on a new line.

Reply:
Oops, right, that was incorrectly formatted by the Rewrap extension I'm using :)